### ISSUE TITLE
(freecad) package file url incorrect

### ISSUE THREAD

**Issue by rhtenhove** [chocolatey-community/chocolatey-packages#1637]
the freecad package is broken, there should be a `-1` appended to the filename.

## Expected Behavior
It should install

## Current Behavior
Install log:

```
You have freecad v0.18.16146 installed. Version 0.19.1 is available based on your source(s).
freecad v0.19.1 [Approved]
freecad package files upgrade completed. Performing other installation steps.
Attempt to get headers for https://github.com/FreeCAD/FreeCAD/releases/download/0.19.1/FreeCAD-0.19.1.a88db11-WIN-x64-installer.exe failed.
The remote file either doesn't exist, is unauthorized, or is forbidden for url 'htt ps://github.com/FreeCAD/FreeCAD/releases/download/0.19.1/FreeCAD-0.19.1.a88db11-WIN-x64-installer.exe'. Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (404) Not Found."
Downloading freecad 64 bit
from 'https://github.com/FreeCAD/FreeCAD/releases/download/0.19.1/FreeCAD-0.19.1.a88db11-WIN-x64-installer.exe'
ERROR: The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://github.com/FreeCAD/FreeCAD/releases/download/0.19.1/FreeCAD-0.19.1.a88db11-WIN-x64-installer.exe' . Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (404) Not Found."
The upgrade of freecad was NOT successful.
```

Package Url: https://chocolatey.org/packages/freecad

## Possible Solution
Run the auto link update bot again? Not sure.

Link should be `https://github.com/FreeCAD/FreeCAD/releases/download/0.19.1/FreeCAD-0.19.1.a88db11-WIN-x64-installer-1.exe` instead of `https://github.com/FreeCAD/FreeCAD/releases/download/0.19.1/FreeCAD-0.19.1.a88db11-WIN-x64-installer.exe`.

## Steps to Reproduce (for bugs)
`choco install freecad -y`

## Context
Just trying to install freecad

## Your Environment
Fails on multiple Windows systems.

**Comment by Squelch** [comment]
The URL for the release package is incorrect.
Chocolatey expects: 'https://github.com/FreeCAD/FreeCAD/releases/download/0.19.1/FreeCAD-0.19.1.a88db11-WIN-x64-installer.exe'
The actual URL is: 'https://github.com/FreeCAD/FreeCAD/releases/download/0.19.1/FreeCAD-0.19.1.a88db11-WIN-x64-installer.exe-1'
Note the appended -1
Chocolatey checks are passing yet the path is obviously incorrect.

**Comment by TheCakeIsNaOH** [comment]
I think this is fixed by #1635, but the package needs to be forced updated.

**Comment by AdmiringWorm** [comment]
@TheCakeIsNaOH it is partially fixed, but a forced update can't be forced yet due to the 32bit edition of not being available for 0.19.1.

As it currently stands, additional changes must be made but it will depend on some decisions.
Either drop support for 32bit completely, and only supply 64bit version in the package (requires changes to the update script as it currently picks up mismatched versions, as well as the install script), or wait and hope that the software authors will provide a 32bit edition (which would mean that the package is broken until that time).
As 32bit is still mentioned as a note to Windows users on their wiki download page (https://wiki.freecadweb.org/Download), I am not completely sure if dropping 32bit support is appropriate.

**Comment by RedBaron2** [comment]
@TheCakeIsNaOH 
If the PR#1638 is approved this package will be updated to the latest version.

@AdmiringWorm
The PR will just silently nix the URL for 32 bit. If the 32 bit version doesn't match the 64 bit version. This is the best practice since all packages are immutable. If need be a fix version can be made to update for a late 32 bit version.

**Comment by rhtenhove** [comment]
This seems to be fixed, I can now update freecad without issue. Thanks!

**Comment by rhtenhove** [comment]
Freecad is on a roll. Now they decided to append just a `1` to the filename, so the download fails again. What would be the solution here?

**Comment by RedBaron2** [comment]
@rhtenhove 
The PR is ready, and after I make the requested changes it should be approved soon. RL required my attention recently, but I should be able to update the PR this upcoming week. Thank you for your patience.

**Comment by rhtenhove** [comment]
@RedBaron2 I was under the assumption it was already fixed for the previous case. My bad. Thanks! I'll keep watch on the PR.

**Comment by sirinsidiator** [comment]
I just tried to install FreeCAD via chocolatey and it looks like the file not found issue still persists. Is there any chance of getting this fixed in the next couple of days or should I just install it the old fashioned way?

**Comment by pauby** [comment]
There are no outstanding PR's for freecad so I'd go ahead and install it the old fashioned way in the meantime.

**Comment by sirinsidiator** [comment]
Will do so. Thanks.

**Comment by RedBaron2** [comment]
@pauby @AdmiringWorm 
The latest PR should fix the file not found error. If someone could run a fix version update for the package. I'm sure the community would be very grateful.

**Comment by pauby** [comment]
@RedBaron2 Which PR?

**Comment by RedBaron2** [comment]
@pauby 
The closed PR #1638

**Comment by RedBaron2** [comment]
@Yrazi 
I'm working on a newer updater for the package that will rectify the problem.

**Comment by RedBaron2** [comment]
I'm working with the FreeCAD developers about how best to remedy this situation. The fix is pending on approval of the developers. I will work on keeping you all informed.

**Comment by AdmiringWorm** [comment]
Closing as fixed, since the changes needed has been merged into the repository.
